### DISCUSSION TITLE
Difficulty using chainlink-brownie

### DISCUSSION THREAD

**Discussion by fiaz-tech** [Cyfrin/foundry-full-course-cu#2560] [Q&A]
I am trying to implement the chainlink-VRF and at the point of installing it with " forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit " in my root directory I'm getting this error  " fatal: not a git repository (or any of the parent directories): .git Error:  git submodule exited with code 128 "

**Comment by EngrPips** [comment]
Hello @fiaz-tech, Please show your `code-editor` as I show mine below.

<img width="1440" alt="Screenshot 2024-09-27 at 10 54 07" src="https://github.com/user-attachments/assets/626b915a-cc5f-4e51-9175-30bfa79db276">

**Reply by fiaz-tech** [reply]
![screen-capture](https://github.com/user-attachments/assets/27c0177a-e34e-4cab-9f9c-a36c27983109)

**Reply by EngrPips** [reply]
if you run `forge init --force` what do you get as a response?

**Reply by fiaz-tech** [reply]
Problem solved, Thank you.

**Reply by EngrPips** [reply]
Awesome, good crushing it.
